InTune/InTune/ViewControllers/ExploreOptionsController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  ExploreOptionsController.swift
+//  InTune
+//
+//  Created by Tiffany Obi on 6/11/20.
+//  Copyright © 2020 Tiffany Obi. All rights reserved.
+//
+
+import UIKit
+
+class ExploreOptionsController: UIViewController {
+    
+    
+    @IBOutlet weak var instrumentsCollectionView: UICollectionView!
+    
+    
+    @IBOutlet weak var genresCollectionView: UICollectionView!
```

Comment:
always follow one line spacing between functions, variables and new classes or structs or imports. Change this to this.

 ```
   @IBOutlet weak var instrumentsCollectionView: UICollectionView!
   @IBOutlet weak var genresCollectionView: UICollectionView!
```